pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
+				return "metadata.name", value, nil
+			case "metadata.name", "metadata.namespace", "status.phase":
```

Comment:
Namespace kind has no namespace field, so shouldn't support the "metadata.namespace" case.